.rubocop.yml

```diff
@@ -35,6 +35,9 @@ Style/NumericLiterals:
 Metrics/MethodLength:
   Enabled: false
 
+Metrics/BlockLength:
+  Enabled: false
```

Comment:
please enable method length and block length metrics
you can skip rubycop validation in spec, db folders if neeeded